lib/cmd_tool_n2ip6.go

```diff
@@ -0,0 +1,63 @@
+package lib
+
+import (
+	"fmt"
+	"github.com/spf13/pflag"
+)
+
+// CmdToolN2IP6Flags are flags expected by CmdToolN2IP6
+type CmdToolN2IP6Flags struct {
+	Help bool
+}
+
+// Init initializes the common flags available to CmdToolN2IP6 with sensible
+func (f *CmdToolN2IP6Flags) Init() {
+	pflag.BoolVarP(
+		&f.Help,
+		"help", "h", false,
+		"show help.",
+	)
+}
+
+// CmdToolN2IP6 converts a number to an IPv6 address
+func CmdToolN2IP6(args []string, printHelp func()) error {
+	if len(args) == 0 {
+		printHelp()
+		return nil
+	}
+
+	expression := args[0]
+	if IsInvalidInfix(expression) {
+		return ErrInvalidInput
+	}
+
+	// n2ip also accepts an expression which is why the following
+	// Steps are being done
+	// Convert to postfix
+	// If it is a single number and not an expression
+	// The tokenization and evaluation would have no effect on the number
```

Comment:
Let's reduce the words on this (and correct command name):

```suggestion
	// NOTE: n2ip6 also accepts an expression, hence the tokenization and evaluation.
```